simple-git/src/lib/plugins/plugin-store.ts

```diff
@@ -1,8 +1,33 @@
-import { SimpleGitPlugin, SimpleGitPluginType, SimpleGitPluginTypes } from './simple-git-plugin';
+import { EventEmitter } from 'node:events';
```

Comment:
heads up that using `node:events` instead of `events` here constituted a breaking change in v3.11, since node 12 doesn't support the `node:` protocol.